### PR TITLE
Clean up code style and remove `.rubocop_todo.yml`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Homebrew/.rubocop.yml [hunk 1]
```diff
@@ -26,9 +25,26 @@ Lint/NestedMethodDefinition:
 Lint/ParenthesesAsGroupedExpression:
   Enabled: true
 
+Metrics/BlockNesting:
+  Max: 5
+
+Metrics/ModuleLength:
+  Max: 360
```
Comment (MikeMcQuaid) [inline Library/Homebrew/.rubocop.yml]: Good idea to just setting these to high values. Mind doing that for line length, too? Would be good to get it down eventually but at least stop it growing for now.

File: Library/Homebrew/cmd/tap.rb [hunk 2]
```diff
@@ -54,8 +54,7 @@ def tap
                     quiet: ARGV.quieter?
       rescue TapRemoteMismatchError => e
         odie e
-      rescue TapAlreadyTappedError, TapAlreadyUnshallowError
-        # Do nothing.
+      rescue TapAlreadyTappedError, TapAlreadyUnshallowError # rubocop:disable Lint/HandleExceptions
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/tap.rb]: Not worth keeping the comment and returning `nil` instead?

File: Library/Homebrew/extend/pathname.rb [hunk 3]
```diff
@@ -186,7 +186,7 @@ def atomic_write(content)
       begin
         tf.chown(uid, gid)
         tf.chmod(old_stat.mode)
-      rescue Errno::EPERM
+      rescue Errno::EPERM # rubocop:disable Lint/HandleExceptions
```
Comment (MikeMcQuaid) [inline Library/Homebrew/extend/pathname.rb]: Worth returning `nil` here and skipping the `disable (and everywhere below)?

**Comment by ilovezfs** [comment]
@reitermarkus maybe should rebase now that #3284 is merged?

**Comment by reitermarkus** [comment]
I'll leave the comments for now. I think it's more obvious to see the difference between explicitly returning `nil` and returning it just to silence RuboCop.

**Pull request merged**